src/sentry/ratelimits/sliding_windows.py

```diff
@@ -139,10 +139,17 @@ def impl(self):
             self._impl = RedisSlidingWindowRateLimiterImpl(self.client)
         return self._impl
 
+    def __disconnect_redis_cluster(self) -> None:
+        if isinstance(self.client, RedisCluster):
+            self.client.disconnect_connection_pools()
+        else:
+            assert isinstance(self.client, StrictRedis)
+            self.client.connection_pool.disconnect()
```

Comment:
oh I meant to replace all the places where this is being added in this PR -- would be a free function imported from `sentry.utils.redis`

Reply:
oh! updated.